EnglishFilter/sections/foreign.txt

```diff
@@ -3584,6 +3584,9 @@ dap-news.com##.ads
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38240
 dispatch.co.kr###topAdRoll
 dispatch.co.kr##body > div[style="height: 45px;"]:not([id]):not([class])
+! https://github.com/AdguardTeam/AdguardFilters/issues/38162
+yazaral11.com##div[id^="sitemap"] > a[href][target="_blank"]
+yazaral11.com##a[href*="btdd3.com"]
```

Comment:
This one not required for me as it is already hidden by previous rule.

Reply:
For me it has another id `bitme`, but it wasn't blocking early with `##`. Now it hides element.
`yazaral11.com###bitme`.
Should I exchanged these rules to:
`yazaral11.com###bitme`
`yazaral11.com##div[id^="sitemap"]`?

Reply:
bitme yes, other one not (too generic).